### PR TITLE
Fix the invite by csv of user already invited

### PR DESCRIPTION
**Pull request by tiagolimpe**
**Feature/Bug description:** When members are invited by csv file and at least one member already been invited, the others are not invited.

**Solution:** Filtering the final list of emails to a list that don't match with the emails of 'sent_invitations' list.

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/institution/managementMembersController.js [hunk 1]
```diff
@@ -413,9 +413,21 @@
             $mdDialog.cancel();
         };
 
+        function removeSentInvitations() {
+            if(selectEmailsCtrl.manageMemberCtrl.sent_invitations.length > 0) {
```
Comment (raonismaneoto) [inline frontend/institution/managementMembersController.js]: if(!_.isEmpty()) is better, isn't it?

File: frontend/institution/managementMembersController.js [hunk 2]
```diff
@@ -413,9 +413,21 @@
...
+                    map(invite => {
+                        return invite.invitee;
+                    });
+                return selectEmailsCtrl.selectedEmails.filter(email => !invitedEmails.includes(email));
+            }
+            return selectEmailsCtrl.selectedEmails;
+        }
+
         selectEmailsCtrl.sendInvite = function sendInvite() {
             if(selectEmailsCtrl.selectedEmails.length > 0 && selectEmailsCtrl.selectedEmails.length <= MAX_EMAILS_QUANTITY) {
```
Comment (raonismaneoto) [inline frontend/institution/managementMembersController.js]: _.isEmpty, _.size

File: frontend/institution/managementMembersController.js [hunk 3]
```diff
@@ -413,9 +413,21 @@
             $mdDialog.cancel();
         };
 
+        function removeSentInvitations() {
```
Comment (raonismaneoto) [inline frontend/institution/managementMembersController.js]: Does this function removes the emails of users that are members too?

File: frontend/institution/managementMembersController.js [hunk 4]
```diff
@@ -413,9 +413,31 @@
...
+
         selectEmailsCtrl.sendInvite = function sendInvite() {
-            if(selectEmailsCtrl.selectedEmails.length > 0 && selectEmailsCtrl.selectedEmails.length <= MAX_EMAILS_QUANTITY) {
-                selectEmailsCtrl.manageMemberCtrl.sendUserInvite(selectEmailsCtrl.selectedEmails);
+            if(!_.isEmpty(selectEmailsCtrl.selectedEmails) && _.size(selectEmailsCtrl.selectedEmails) <= MAX_EMAILS_QUANTITY) {
+                var emails = removePendingAndMembersEmails();
+                if(!_.isEmpty(emails)) {
+                    selectEmailsCtrl.manageMemberCtrl.sendUserInvite(emails);
+                } else {
+                    MessageService.showToast("Todos os e-mails selecionados já foram convidados ou pertencem a algum membro da instituição.")
```
Comment (mayzabeel) [inline frontend/institution/managementMembersController.js]: deixa só "e-mails selecionados já foram convidados ou pertencem a algum membro da instituição."

File: frontend/institution/managementMembersController.js [hunk 5]
```diff
@@ -413,9 +413,31 @@
             $mdDialog.cancel();
         };
 
+        function removePendingAndMembersEmails() {
```
Comment (mayzabeel) [inline frontend/institution/managementMembersController.js]: falta testar

**Pull request merged**